tests/test_cli.py

```diff
@@ -868,3 +868,19 @@ def test_invoke_invalid_command(runner, tmpdir):
     assert (
         'Error: Invalid setting for [main][default_command]' in result.output
     )
+
+
+def test_show_priority(runner, todo_factory, todos):
+    todo = todo_factory(summary='harhar\n', priority=1)
+
+    result = runner.invoke(cli, ['show', '1'])
+    assert '1' in result.output
```

Comment:
I see those tests passing, but you have [style errors](https://travis-ci.org/pimutils/todoman/jobs/226445642#L215-L216).

You can run `tox -e flake8` to test these locally.